### ISSUE TITLE
Reboot recovery after full wipe doesn't work on bullhead 3.1.0-0

### ISSUE THREAD

**Issue by mew1033** [TeamWin/Team-Win-Recovery-Project#890]
To reproduce:

1. Go to TWRP
2. In wipe menu, select advanced wipe and check all the boxes
3. Swipe to Wipe
4. Go Home
5. Go to reboot menu and hit recovery
6. Boot hangs on Google screen.

**Comment by andi34** [comment]
Can confirm on tab2 and maguro.

On tab2 I fixed it by reverting https://gerrit.omnirom.org/#/c/22218/1

**Comment by blastagator** [comment]
Can you get back into TWRP by using the hardware key combo?  Or after doing these steps it breaks indefinitely and there is no way to get back in?

**Comment by mew1033** [comment]
Yes, hardware key combo works fine. Reboot back to bootloader and use volume/power keys to get to recovery.
Rebooting to recovery also works from inside the OS.

**Comment by ghost** [comment]
Can confirm this condition also exists on flounder using TWRP 3.1.0.0

The steps to reproduce are exactly as outlined above. You have to use the hardware buttons to boot into the bootloader and from there you can boot into recovery.
